package-lock.json

```diff
@@ -8560,231 +8560,6 @@
       "resolved": "configs/webpack-config-compass",
       "link": true
     },
-    "node_modules/@mongosh/arg-parser": {
```

Comment:
I don't know why, but this is once again un-hoisting the shell dependencies for some reason.

So normally I just do

```sh
npm i mongodb@latest @mongosh/logging@latest @mongosh/node-runtime-worker-thread@latest @mongosh/browser-repl@latest bson@latest @mongodb-js/devtools-connect@latest @mongodb-js/devtools-proxy-support@latest && git checkout -- package.json && npm i
```

now

Reply:
Thanks! I didn't even notice that. Trying again..